src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -143,6 +144,25 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     s"docker run --rm -v ${backendCall.callRootPath.toAbsolutePath}:$callPath -i $image"
   }
 
+  /**
+    * If using Mac OS X with Docker Machine, by default Cromwell can only use the -v flag to 'docker run'
+    * if running from somewhere within the user's home directory.  If not, then -v will mount to the
+    * virtual machine that Docker Machine creates.  This ends up bubbling up to Cromwell when it can't find
+    * an RC file in the call's output directory (because it's actually on the virtual machine).
+    *
+    * This function will at least detect this case and log some WARN messages.
+    */
+  private def detectDockerMachinePossibleMisusage(backendCall: BackendCall, logger: WorkflowLogger) = {
+    val os = Option(System.getProperty("os.name"))
+    val homeDir = Option(System.getProperty("user.home")).map(Paths.get(_).toAbsolutePath.toString)
+    val callRoot = backendCall.callRootPath.toAbsolutePath.toString
+    val dockerMachineName = sys.env.get("DOCKER_MACHINE_NAME")
+    if (dockerMachineName.nonEmpty && os.contains("Mac OS X") && homeDir.isDefined && !callRoot.startsWith(homeDir.get)) {
+      logger.warn(s"You are running Docker using Docker Machine and your working directory is not a subdirectory of ${homeDir.get}")
+      logger.warn(s"By default, Docker Machine only shares your home directory, which means Cromwell can only be run from your home directory")
```

Comment:
Cromwell must be run under the home directory, not necessarily in the home directory